nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
Good catch. How would you like this to be implemented? It's straightforward to just add `scale` to `propose_point` and keep the functionality the same as the original class. Otherwise, I think the proper thing to do would be to adjust the acceptance ratio as already implemented, then once an "optimal" scale has been reached re-run the MCMC with `scale` fixed.